app/grandchallenge/core/error_handlers.py

```diff
@@ -0,0 +1,98 @@
+from grandchallenge.notifications.models import Notification, NotificationType
+
+
+class ErrorHandler:
+    def handle_error(self, *, error_message, user=None, interface=None):
+        raise NotImplementedError("Subclasses must implement this method")
+
+
+class JobCIVErrorHandler(ErrorHandler):
+
+    def __init__(self, job):

```

Comment:
All of the `__init__` definitions in this file are missing `*args`, `**kwargs` and calls to `super()`. We can future-proof the codebase a little by adding that, which would also add out stylistic preference for kwargs only methods.
```suggestion
    def __init__(self, *args, job, **kwargs):
        super().__init__(*args, **kwargs)

```

Reply:
3 more instances of this in this file.